### ISSUE TITLE
npm start fails

### ISSUE THREAD

**Issue by matthiasbeyer** [anderejd/electron-wasm-rust-example#8]
I'm a complete noob, so take a grain of salt.

When `npm start`ing, I get:

```
App threw an error during load
TypeError: protocol.registerStandardSchemes is not a function
    at Object.<anonymous> (/home/m/archive/development/rust/distrox/main.js:58:10)
    at Module._compile (internal/modules/cjs/loader.js:808:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:820:10)
    at Module.load (internal/modules/cjs/loader.js:677:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:609:12)
    at Function.Module._load (internal/modules/cjs/loader.js:601:3)
    at loadApplicationPackage (/nix/store/nd6cx53vcq2ai2l0nlss3qqzl52xfhjk-electron-5.0.0/lib/electron/resources/default_app.asar/main.js:105:12)
    at Object.<anonymous> (/nix/store/nd6cx53vcq2ai2l0nlss3qqzl52xfhjk-electron-5.0.0/lib/electron/resources/default_app.asar/main.js:153:5)
    at Module._compile (internal/modules/cjs/loader.js:808:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:820:10)
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError: protocol.registerStandardSchemes is not a function
    at Object.<anonymous> (/home/m/archive/development/rust/distrox/main.js:58:10)
    at Module._compile (internal/modules/cjs/loader.js:808:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:820:10)
    at Module.load (internal/modules/cjs/loader.js:677:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:609:12)
    at Function.Module._load (internal/modules/cjs/loader.js:601:3)
    at loadApplicationPackage (/nix/store/nd6cx53vcq2ai2l0nlss3qqzl52xfhjk-electron-5.0.0/lib/electron/resources/default_app.asar/main.js:105:12)
    at Object.<anonymous> (/nix/store/nd6cx53vcq2ai2l0nlss3qqzl52xfhjk-electron-5.0.0/lib/electron/resources/default_app.asar/main.js:153:5)
    at Module._compile (internal/modules/cjs/loader.js:808:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:820:10)
Fontconfig warning: "/etc/fonts/fonts.conf", line 86: unknown element "blank"
```

**Comment by matthiasbeyer** [comment]
Removing the `protocol.registerStandardSchemes()` call from main.js works.

But now I get 

```
distrox.js:167 Fetch API cannot load app://./wasm/..._bg.wasm. URL scheme must be "http" or "https" for CORS request.
```

in the dev console...

**Comment by matthiasbeyer** [comment]
After adding 

```
protocol.registerSchemesAsPrivileged([
  { scheme: 'http', privileges: { bypassCSP: true } }
])
```

in `main.js`, it cannot find the file (the `_bg` file).

**Comment by matthiasbeyer** [comment]
Seems that 

```
   protocol.registerSchemesAsPrivileged([
     { scheme: 'app', privileges: { bypassCSP: true, supportFetchAPI: true } }
   ])
```

does the trick.

**Comment by anderejd** [comment]
Thanks for the report!

I'll have to try this myself later, maybe something has changed with electron, reopening.

**Comment by anderejd** [comment]
Can confirm, this error happens when I upgrade to the latest electron.

**Comment by anderejd** [comment]
Fixed by cdc711064d53e499411dc60e2d1dbd22cdbe5279

Seems like the problem was that `protocol.registerStandardSchemes` was deleted in electron 5.
